src/components/Capture/index.js

```diff
@@ -1,243 +1,36 @@
 import { h, Component } from 'preact'
-import { events } from 'onfido-sdk-core'
-import classNames from 'classnames'
-import randomId from '../utils/randomString'
-import { Uploader } from '../Uploader'
-import Camera from '../Camera'
-import Confirm from '../Confirm'
-import { FaceTitle } from '../Face'
-import { DocumentTitle } from '../Document'
-import isDesktop from '../utils/isDesktop'
-import DetectRTC from 'detectrtc'
-import style from './style.css'
-import { functionalSwitch, impurify } from '../utils'
-import { canvasToBase64Images } from '../utils/canvas.js'
-import { fileToBase64, isOfFileType, fileToLossyBase64Image } from '../utils/file.js'
+import Capture from './capture.js'
+import { impurify } from '../utils'
 
-const StatelessDocumentCapture = options =>
-  <Capture method='document' autoCapture={true} {...options} />
+const StatelessFrontDocumentCapture = options =>
```

Comment:
I agree, but they need a different name to the one we export. I thought 'stateless' was the most descriptive way to differentiate them from the public exported names